ffi/custom_passes.cpp

```diff
@@ -685,13 +693,15 @@ struct RefPrunePass : public FunctionPass {
         // RAII push cur_node onto the work stack
         raiiStack<SmallVectorImpl<BasicBlock*>> raii_path_stack(path_stack, cur_node);
 
+        unsigned subgraph_size = 0;
```

Comment:
Could do with a comment about use, that it's pass by ref accumulator?

Reply:
fixed in https://github.com/numba/llvmlite/pull/644/commits/fa6092b8de364fe7a04c00a08523cc1c1e027783